flashstep.lua

```diff
@@ -0,0 +1,20 @@
+--Teleports unit to cursor
+function flashstep(flash,step)
+	local flash = df.global.world.units.active[0]
+	local step
+		if df.global.ui_advmode.menu == df.ui_advmode_menu.Look then
+			step = df.global.cursor
+		else
+			qerror("No [l] cursor located!  You kinda need it for this script.")
+		end
+	local unitoccupancy = dfhack.maps.getTileBlock(flash.pos).occupancy[flash.pos.x%16][flash.pos.y%16]
```

Comment:
Most of those like that are due to pulling bits of other scripts over to convert for a different use.